sass/layout/_markdown.scss

```diff
@@ -266,7 +266,7 @@ h6 {
         background: $white;
 
         &:nth-child(2n) {
-            background-color: #f8f8f8;
+            background: rgba(var(--center-channel-color), 0.07);
```

Comment:
Hey @galihmelon, please use the `--center-channel-color-rgb` variable.

Reply:
Hey @asaadmahmood, the var has been changed now.